examples/seismic/acoustic/acoustic_example.py

```diff
@@ -23,34 +23,23 @@ def smooth10(vel, shape):
 
 def acoustic_setup(dimensions=(50, 50, 50), spacing=(15.0, 15.0, 15.0),
                    tn=500., time_order=2, space_order=4, nbpml=10, **kwargs):
-
-    ndim = len(dimensions)
     nrec = dimensions[0]
-    origin = tuple([0.]*ndim)
-    spacing = spacing[:ndim]
-
-    # Velocity model, two layers
-    true_vp = np.ones(dimensions) + .5
-    true_vp[..., int(dimensions[-1] / 3):dimensions[-1]] = 2.5
-
-    # Define seismic data
-    model = Model(origin, spacing, dimensions, true_vp, nbpml=int(nbpml))
+    model = demo_model('layers', ratio=3, shape=dimensions,
+                       spacing=spacing, nbpml=nbpml)
 
     # Derive timestepping from model spacing
-    dt = model.critical_dt
-    if time_order == 4:
-        dt *= 1.73
+    dt = model.critical_dt * 1.73 if time_order == 4 else 1.0
```

Comment:
Not sure what the if does, apply to the * only or to dt?

If dt, should be `else model. critical_dt`